scripts/common.sh

```diff
@@ -58,12 +58,12 @@ get_minor_version() {
 }
 get_patch_version() {
   local release_tag="$1"
-  get_release_version "$release_tag" | awk -F'.' '{print $3}'
+  get_release_version "$release_tag" | awk -F'[.-]' '{print $3}'
 }
 
 validate_version() {
   local version="$1"
-  if [[ ! $version =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]
+  if [[ ! $version =~ ^[0-9]+\.[0-9]+\.[0-9]+(-.*)?$ ]]
```

Comment:
The `-alpha` qualifier is scary, I'm afraid that it'll break something in the release script.

Reply:
> How so?

Because the previous version numbers forever didn't have the suffix, and there's some expectation in the conventions like `1.2.3` is different from `2.0.0-alpha` because it contains alphabetic chars.

We absolutely need those changes, and a few more.....